pkg/apis/iam/v1beta1/krm_types.go

```diff
@@ -62,6 +74,14 @@ type MemberReference struct {
 	Name      string `json:"name"`
 }
 
+type BigQueryConnectionConnectionMemberReference struct {
```

Comment:
Suggest defining this struct in BQCC API to avoid cyclic dependencies and make it reusable.

Reply:
Done!